flink-runtime/src/test/java/org/apache/flink/runtime/state/ChannelPersistenceITCase.java

```diff
@@ -76,17 +67,17 @@ public void testReadWritten() throws Exception {
 			singletonMap(resultSubpartitionInfo, resultSubpartitionInfoData)
 		);
 
-		assertArrayEquals(inputChannelInfoData, read(
```

Comment:
It looks like some cleanup is needed here.

Reply:
Right, forgot about this test :)